### DISCUSSION TITLE
[Community] Starstruck achievements

### DISCUSSION THREAD

**Discussion by BitTheCat** [Schweinepriester/github-profile-achievements#150] [General]
Hi all, 
for get starstruck need to reach 16 star. If u want u can put star on my readme repository, I will do the same for you.

Add a answer with your profile repository for have a chance to get the star from other user.

* [BitTheCat profile repository](https://github.com/BitTheCat/bitthecat)

**Comment by Keyrxng** [comment]
Return the favor troops ;) 

https://github.com/Keyrxng/testnet-wallet-drainer

**Reply by number0x0001** [reply]
Done. Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by 404jv** [reply]
I did it. Please star this https://github.com/404jv/dev-landia

**Reply by Crushoverride007** [reply]
Done. - please leave one here https://github.com/Crushoverride007/Crushoverride007 
Join the TROOPS to hunt them all

**Reply by faintout** [reply]
Done. Please leave a star here:
https://github.com/faintout/ThreeRoomDemo

**Reply by faintout** [reply]
> https://github.com/pingshunhuangalex/rime-keydo

Done. Please leave a star here:
https://github.com/faintout/ThreeRoomDemo

**Comment by number0x0001** [comment]
It has been done. Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by wYaobiz** [reply]
Done. Thank you! 
[My Repo](https://github.com/wYaobiz/Full-Stack-Web-Developer-Roadmap-2018)

**Reply by 404jv** [reply]
I did it. Please star this https://github.com/404jv/dev-landia

**Comment by wYaobiz** [comment]
Done. Thank you! 
[My Repo](https://github.com/wYaobiz/Full-Stack-Web-Developer-Roadmap-2018)

**Reply by number0x0001** [reply]
Starred. Please return it on https://github.com/number0x0001/Seanux-Linux-Files-Scripts

Thanks!

**Reply by 404jv** [reply]
Done, please star this https://github.com/404jv/dev-landia

**Reply by Crushoverride007** [reply]
done. please leave one here https://github.com/Crushoverride007/Crushoverride007 @wYaobiz

**Reply by wYaobiz** [reply]
All above are done, pls check

**Reply by nel0x** [reply]
Did all above. Please star https://github.com/nel0x/arch-installer

**Comment by 404jv** [comment]
Done! My repo is https://github.com/404jv/dev-landia

Thanks

**Reply by nel0x** [reply]
Done. Please star https://github.com/nel0x/arch-installer

**Comment by Crushoverride007** [comment]
Done @404jv join the troops https://github.com/Crushoverride007/Crushoverride007

**Reply by 404jv** [reply]
Thanks. I joined!

**Reply by number0x0001** [reply]
Please leave on here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Comment by Luscarvalho** [comment]
Return the favor ;)

https://github.com/Luscarvalho/Luscarvalho

**Reply by number0x0001** [reply]
Please leave on here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Comment by nel0x** [comment]
Did all above. Please star https://github.com/nel0x/arch-installer :)

**Reply by number0x0001** [reply]
Please leave on here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Comment by melvincwng** [comment]
For those who are interested to help ⭐:

1. https://github.com/melvincwng/facebook-clone
2. https://github.com/Your-First-Open-Source-Project/Your-First-Open-Source-Project

Thanks In advance, glad to see such a vibrant and inclusive community here on GitHub 😊
